src/main/java/io/github/perrymant/moneymaker/MoneyFormatter.java

```diff
@@ -0,0 +1,15 @@
+package io.github.perrymant.moneymaker;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+class MoneyFormatter {
+    private static final Locale UK = Locale.UK;
+
+    static String formatPounds(int amount) {
```

Comment:
There's no need for this to be `static`. Avoid static whenever possible as it makes is hard to mock out in tests, should we ever need to.